src/stream/src/executor/aggregation/approx_count_distinct.rs

```diff
@@ -27,94 +27,155 @@ use risingwave_common::types::{Datum, DatumRef, Scalar, ScalarImpl};
 use super::StreamingAggStateImpl;
 use crate::executor::error::StreamExecutorResult;
 
-const INDEX_BITS: u8 = 14; // number of bits used for finding the index of each 64-bit hash
+const INDEX_BITS: u8 = 16; // number of bits used for finding the index of each 64-bit hash
 const NUM_OF_REGISTERS: usize = 1 << INDEX_BITS; // number of registers available
 const COUNT_BITS: u8 = 64 - INDEX_BITS; // number of non-index bits in each 64-bit hash
 
 // Approximation for bias correction for 16384 registers. See "HyperLogLog: the analysis of a
 // near-optimal cardinality estimation algorithm" by Philippe Flajolet et al.
 const BIAS_CORRECTION: f64 = 0.72125;
```

Comment:
seems to be 0.721288 for `INDEX_BITS = 16`.

Probably we should initialize it by the formula within `lazy_static`

Reply:
Actually,it can be evaluated as const